hosted/gke/helper/helper_cluster.go

```diff
@@ -151,6 +155,18 @@ func CreateGKEClusterOnGCloud(zone string, clusterName string, project string, k
 	labels := GetLabels()
 	labelsAsString := k8slabels.SelectorFromSet(labels).String()
 
+	// creating GKE using gcloud changes the kubeconfig to use GKE; this can be problematic for test cases that need to use local cluster;
```

Comment:
This could be the case for AKS, EKS as well. Better have function in helper_common.go

Reply:
You're right. I did not make it a common function earlier because I was waiting to check if it a case for AKS and EKS as well. Is it fine if I implement this in later iterations?

Reply:
Please add a TODO